docs/index.rst

```diff
@@ -1,14 +1,38 @@
-###########################
-Qiskit Nature Documentation
-###########################
+######################
+Qiskit Nature overview
+######################
+
+Overview
+==============
+
+**Qiskit Nature** is an open-source framework that supports problems including ground state energy computations,
+excited states and dipole moments of molecule, both open and closed-shell.
+
+The code comprises chemistry drivers, which when provided with a molecular
+configuration will return one and two-body integrals as well as other data that is efficiently
+computed classically. This output data from a driver can then be used as input in Qiskit
+Nature that contains logic which is able to translate this into a form that is suitable
```

Comment:
I'm not yet too happy with the phrasing of this sentence but also don't have an immediate suggestion on how to improve it.. :thinking:

Reply:
How about something like the following:
> Qiskit Nature provides logic to interpret this output data, translating it into a form that is suitable for quantum algorithms.